logstash/tasks/es-config.yml

```diff
@@ -0,0 +1,6 @@
+- name: Configure kibana
```

Comment:
If default configuration file changed with custom then first rename default one 
for ex. change it with logstash.conf.example

Reply:
done